<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>存款计算器</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QLabel" name="labYears">
      <property name="text">
       <string>年限</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labMoneyCurrent">
      <property name="text">
       <string>本金</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="txtRate">
      <property name="text">
       <string>0.04125</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtMoneyCurrent">
      <property name="text">
       <string>1000000</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labRate">
      <property name="text">
       <string>利率</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labYear">
      <property name="text">
       <string>期限</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labType">
      <property name="text">
       <string>方式</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cboxType">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>单利</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>复利</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cboxYear">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>1年</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3年</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5年</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="txtYears">
      <property name="text">
       <string>30</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="txtMoneyAll"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labMoneyAll">
      <property name="text">
       <string>总计</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="3">
     <widget class="QPushButton" name="btnOk">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>计算</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
